kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionEditorViewImpl.java

```diff
@@ -462,46 +504,6 @@ private String getTypeRef() {
         return qName.getLocalPart();
     }
 
-    private void setExpressionNameAndDataType(final ExpressionProps expressionProps) {
-        setExpressionName(expressionProps.name);
-        setTypeRef(expressionProps.dataType);
-        updateCanvasNodeName();
-    }
-
-    private void setExpressionName(final String name) {
-        final HasName fallbackHasName = hasExpression instanceof HasName ? (HasName) hasExpression : HasName.NOP;
-        hasName.orElse(fallbackHasName).setName(new Name(name));
-    }
-
-    @SuppressWarnings("unchecked")
-    private void setTypeRef(final String dataType) {
-        final QName typeRef = BuiltInTypeUtils
-                .findBuiltInTypeByName(dataType)
-                .orElse(BuiltInType.UNDEFINED)
-                .asQName();
-        if (hasExpression instanceof HasVariable) {
-            ((HasVariable<InformationItemPrimary>) hasExpression).getVariable().setTypeRef(typeRef);
-        } else if (hasExpression.getExpression() != null && hasExpression.getExpression().asDMNModelInstrumentedBase().getParent() instanceof HasVariable) {
-            ((HasVariable<InformationItemPrimary>) hasExpression.getExpression().asDMNModelInstrumentedBase().getParent()).getVariable().setTypeRef(typeRef);
-        }
-    }
-
-    @SuppressWarnings("rawtypes")
-    private void updateCanvasNodeName() {
-        final AbstractCanvasHandler canvasHandler = (AbstractCanvasHandler) sessionManager.getCurrentSession().getCanvasHandler();
-        final CompositeCommand.Builder<AbstractCanvasHandler, CanvasViolation> commandBuilder = new CompositeCommand.Builder<>();
-        final Element element = canvasHandler.getGraphIndex().get(nodeUUID);
-        if (element.getContent() instanceof Definition) {
-            final Definition definition = (Definition) element.getContent();
```

Comment:
Can you please help me to understand how these methods are mapped with the new implementation?